CHANGELOG.md

```
@@ -1,5 +1,19 @@
 ## CHANGE LOG
 
+Version 3.7.0 *(6 February 2026)*
```

markdownlint-cli2 (0.20.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)